x-pack/plugins/security_solution/public/common/components/tables/helpers.tsx

```diff
@@ -219,6 +239,8 @@ interface RowItemOverflowProps {
   maxOverflowItems: number;
   overflowIndexStart: number;
   rowItems: string[];
+  fieldType?: string;
```

Comment:
Should these be optional considering we're defaulting the values so they should always be defined? Just want to prevent any errors in the future by someone accidentally removing these

Reply:
removed the default values